config/redis.yml

```diff
@@ -92,6 +92,9 @@ development: &defaults
   va_mobile_session:
     namespace: va-mobile-session
     each_ttl: 855
+  login_redirect_application:
+    namespace: login-redirect-application
+    each_ttl: 300 # 5 minutes
```

Comment:
@patrickvinograd thinking through this TTL, would we want to give the user more than 5 minutes to login?  Is it possible that a signup process, with verification procedures will take longer than this?

Reply:
Good call.  5 minutes is way to short for a first-time sign up. After watching a user research session with a blind veteran who used a screen reader who needed to both verify his email address and set up 2fa I'd say minimum of 30 minutes, but that 1h would be perfectly reasonable.

Reply:
thanks Anna, just increased this to 1 hour